lambda_src/drivers/process_https.py

```diff
@@ -116,21 +133,39 @@ def process_row(
     next_url: Optional[str] = req_url
     row_data: List[Any] = []
 
+    LOG.info('Making requests to host: %s', req_host)
     LOG.debug('Starting pagination.')
     while next_url:
-        LOG.debug(f'next_url is {next_url}.')
+        LOG.info('next_url is %s.', next_url)
         req = Request(next_url, method=req_method, headers=req_headers, data=req_data)
         links_headers = None
 
         try:
             LOG.debug(f'Making request with {req}')
             res = urlopen(req)
+
+            req_obj_bytes = (req.full_url + str(req.headers) + str(req.data)).encode()
+
+            LOG.info(
+                'Request sent with size: %d bytes, type: %s and sha256: %s, for URL: %s.',
+                len(req_obj_bytes),
+                req_method,
+                sha256(req_obj_bytes).hexdigest(),
+                next_url,
+            )
+
             links_headers = parse_header_links(
                 ','.join(res.headers.get_all('link', []))
             )
             response_headers = dict(res.getheaders())
+            response_status = res.status
             res_body = res.read()
-            LOG.debug(f'Got the response body with length: {len(res_body)}')
+            LOG.info(
+                'Got the response with status code: %d and body size: %d bytes, for URL: %s.',
+                response_status,
+                len(res_body),
```

Comment:
use res.body/similar instead

Reply:
It doesn't like the result object has an attribute like that

Reply:
we should also include how long the request took into a separate info log

we should also respond with the time the request / response took into the metadata returned when the user turns `verbose` option on